models/llm_layer/llama_layer_prototype.py

```diff
@@ -0,0 +1,105 @@
+# labels: name::llama_7b_cache_layer author::transformers task::Natural_Language_Processing
```

Comment:
Should LLaMa fall under `Generative_AI` tag?
Or we should start supporting multiple tags per model.

Reply:
https://github.com/groq/mlagility/pull/317/commits/4dc27894b0f92739d9dafdfaa8031f39b42cb9ba